tests/project-type/workspace-instance.spec.ts

```diff
@@ -1,24 +1,20 @@
-import { WorkspaceImpl } from "@sap/project-api";
 import { expect } from "chai";
-import { uniq } from "lodash";
-
+import * as _ from "lodash";
```

Comment:
**subjective**

while this does not matter as much for test code it is better to import only the names you need
`import { map, range }  from "lodash";`

Reasons:
- bundling tools can better do tree- shaking 
   - https://webpack.js.org/guides/tree-shaking/
- the identifier `_` (underscore) is useful as an ID for very short lambda functions